webserver/views/api/v1/core.py

```diff
@@ -109,3 +111,48 @@ def _validate_offset(offset):
     else:
         offset = 0
     return offset
+
+@bp_core.route("/low-level", methods=["POST"])
+def submit_low_level_nombid():
```

Comment:
This looks pretty good. a few comments:
 * Perhaps we can name the url endpoint to say that it's specifically one to generate an id for data with no mbid
 * I think that a submission to this endpoint with a musicbrainz tag should be an error. It can be up to the client to see if a data file has an mbid and use the correct endpoint
 * If the data already exists (the md5 is already in the database), I wonder if we can report this somehow:
  * Use HTTP 201 CREATED if we make it, and 200 otherwise
  * Redirect to the existing item if it's already there (will require changing the client to know how to understand the redirect)